stellar/deploy-contract.js

```diff
@@ -86,7 +62,7 @@ async function processCommand(options, config, chain) {
         chain.contracts = {};
     }
 
-    const cmd = `soroban contract deploy --wasm ${wasmPath} --source ${options.privateKey} --rpc-url ${rpc} --network-passphrase "${networkPassphrase}"`;
+    const cmd = `stellar contract deploy --wasm ${wasmPath} --source ${options.privateKey} --rpc-url ${rpc} --network-passphrase "${networkPassphrase}"`;
```

Comment:
```suggestion
    const cmd = `${stellarCmd} contract deploy --wasm ${wasmPath} --source ${options.privateKey} --rpc-url ${rpc} --network-passphrase "${networkPassphrase}"`;
```
create a constant for this maybe?

Reply:
updated.